src/instructlab/configuration.py

```diff
@@ -361,6 +361,12 @@ class _evaluate(BaseModel):
     gpus: Optional[int] = Field(
         default=None, description="Number of GPUs to use for running evaluation."
     )
+    results: Optional[str] = Field(
```

Comment:
Would format be a better term?

Reply:
Originally I actually did use `format` but I got a MyPy error about overriding built-ins

Reply:
It might be worth it to squash the error.  Unless there is another commonly used variation other than format.

Reply:
Will do :+1: